lib/constants/validators.dart

```diff
@@ -54,3 +56,17 @@ Function(String) bloodGroupValidator = (String bloodGroup) {
 
   return null;
 };
+
+
+Function(String) emailAddressValidator = (String emailaddress) {
+  String email = r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]";
+  bool valids = RegExp(email).hasMatch(emailaddress);
+
+  if (emailaddress.isEmpty) {
+    return "Please enter a valid email";
+  }
+  if (valids == false) {
```

Comment:
It can be now written as `!isValid`

Reply:
if is write isValid == false. is it wrong ?????

Reply:
No, it is not wrong. But it is not the best way to write this.